README.md

```diff
@@ -79,15 +79,18 @@ This includes the creation of a configuration file called `env.json`. Complete t
     `CANVAS_URL` | The Canvas instance URL to be used as the base URL for API requests that use the `CANVAS TOKEN`.
     `CANVAS_TOKEN` | The Canvas token used for authenticating to the API when not using the U-M API Directory.
     `NUM_ASYNC_WORKERS` | Number of workers for asynchronous API calls; the default is 8.
-    `ZOOM_BASE_URL` | The base URL for calls to the Zoom API.
-    `ZOOM_TOKEN` | The token for authenticating to the Zoom API.
-    `ZOOM_EARLIEST_FROM` | The earliest date to retrieve zoom content from.
     `DEFAULT_SLEEP_TIME` | Amount of time to sleep between re-tries when given a 429 error and no Retry-After in the response headers.
     `WAREHOUSE_INCREMENT` | A 17-digit number that can be added to some Canvas IDs to create Unizin Data Warehouse IDs (not in use currently).
     `UDW` | An object containing the necessary credential information for connecting to the Unizin Data Warehouse, where data will be pulled from.
     `CREATE_CSVS` | A boolean (`true` or `false`) indicating whether CSVs should be generated by the execution.
     `INVENTORY_DB` | An object containing the necessary credential information for connecting to a MySQL database, where output data will be inserted.
     `APPEND_TABLE_NAMES` | An array of strings identifying tables that accumulate data and from which records should never be dropped programmatically.
+    `ZOOM_CONFIG` | An object containing the configuration for Zoom. You need a key for each object, this key is inserted into the DB (see below)
+    **ZOOM_CONFIG** |
+    `BASE_URL` | The base URL for calls to the Zoom API.
+    `API_KEY` | The API key to get the JWT token.
+    `API_SECRET` | The API secret to get the JWT token.
+    `EARLIEST_FROM` | The earliest date to retrieve zoom content from.
```

Comment:
This looks okay for now. I suggested to @lsloan that we add another column for the nested level, but that doesn't work great with your key setup. Could you move this back up with the `SLEEP` param? Just trying to keep the data fetching stuff together. Also, could `EARLIEST_FROM` apply globally to all `ZOOM_CONFIG`s?

Reply:
Yeah I'm not sure how best to format what's essentially needing to be a nested table. 

EARLIEST_FROM could apply globally but I actually think that would be more work, and I can see wanting the ability to configure these independently.

Reply:
Fair enough on the `EARLIEST_FROM`, just thought I'd ask. Seemed to me like we'd want them to be in sync, and keeping them separate might introduce possible inconsistencies.

Reply:
I feel like the DEFAULT_SLEEP_TIME could be used by other API calls too so it's not really specific to Zoom. I'm not sure how it would look since these values only apply to Zoom. Hmm. . .

Reply:
I'm thinking `ZOOM_EARLIEST_FROM`.

Reply:
Well if it's in the block EARLIEST_FROM under Zoom makes sense. If we had an EARLIEST_FROM apply to all configs then we could put it on the top level of ZOOM_CONFIG and still call it the same, but this seems fine and is going to change when we only have to query just the updated data too.

Reply:
Still would appreciate if this got moved up to where it was.

Reply:
Don't you think it would be confusing? I don't think with Markdown we can add documentation for json objects and lists all in the same table as additional rows and not have it be confusing where those values apply to. I've haven't found any great example of documenting json in Markdown and if it does exist it won't be as a single table. I think anything this would involve us moving to readthedocs.io and having a Sphinx style documentation. 

I don't believe JSON makes a good configuration and I also don't think we should have this documention taking up space in the README (even if we're the only people looking at it). I feel like I've spent too much time trying to figure out a way to make this look good and understandable.

Reply:
Okay, don't worry about it anymore then, it's fine like it is for now, we can revisit when the change to YAML happens.

Reply:
Yeah, in looking up the best way to do this I also saw a some negatives about YAML so that may need more research too. :)